src/oss/langchain/multi-agent.mdx

```diff
@@ -298,13 +299,21 @@ const callSubagent1 = tool(
 
 ## Handoffs
 
-In **handoffs**, agents can directly pass control to each other. The “active” agent changes, and the user interacts with whichever agent currently has control.
+In **handoffs**, agents pass control to each other. There are two main approaches:
 
-Flow:
+| Approach | How it works | Best for |
+|----------|--------------|----------|
+| [**Tool handoffs**](#tool-handoffs) | Agent calls a tool that updates state to transfer control | Dynamic routing, user-driven flows |
+| [**Graph handoffs**](#graph-handoffs) | Graph edges define transitions between agents | Fixed pipelines, deterministic routing |
```

Comment:
One of the points of handoffs was that it allows users to maintain conversation with the active agent. Graph handoffs definitely do not though that, so we should find a different way of presenting it and likely not grouping together.

----

We probably want to introduce an idea of a **router** that routes the user's input into the agent that's carrying the conversation (if there are multiple agents) or suggest an architecture where there's a single agent that's doing all the hard work (via skills)